docworks-dts/lib/multiple-files/repoCreator.js

```diff
@@ -0,0 +1,50 @@
+const { createHierarchicalServicesMap } = require('./serviceUtils')
+const { moduleBuilder, $wGlobalNamespaceBuilder } = require('./builders')
+const { $W_NAME } = require('./constants')
+const { getModulesDependenciesMap } = require('./providers/modulesDependencies')
+const {
+	emit,
+	dtsImportDefault
+} = require('./generators')
+
+const is$w = name => name === $W_NAME
+
+const createModulesFilesMap = ({ services, run$wFixer }) => {
+	const servicesMap = createHierarchicalServicesMap(services)
+	return Object.keys(servicesMap).reduce((filesMap, moduleName) => {
+		const rootService = servicesMap[moduleName]
+		return {
+			...filesMap,
+			[moduleName]: is$w(moduleName)
+				? $wGlobalNamespaceBuilder(rootService, { run$wFixer })
+				: moduleBuilder(rootService)
+		}
+	}, {})
+}
+
+const createImportStatements = dependencies => {
+	return Object.keys(dependencies).map(depModuleKey => {
+		const dependency = dependencies[depModuleKey]
+		return emit(dtsImportDefault(dependency.to, dependency.from))
+	})
+}
+
+const enrichModuleContent = (module, extraConetnt) => {
+	if (module) {
+		module.content = [...extraConetnt, module.content].join('\n')
+	}
+}
+
+const repoCreator = ({ services, run$wFixer }) => {
+	const modulesFiles = createModulesFilesMap({ services, run$wFixer })
+
+	const modulesDependencies = getModulesDependenciesMap()
+	Object.keys(modulesDependencies).forEach(moduleKey => {
+		const dependencies = modulesDependencies[moduleKey]
+		const importStatments = createImportStatements(dependencies)
+		enrichModuleContent(modulesFiles[moduleKey], importStatments)
```

Comment:
I like the word `enrich` 😅 
But I think here you can be more specific, you're basically prepending content.
So maybe `prependModuleContent`.

Also, there's just one use case for this function, which is to prepend import statements. So you can be even more specific and call it `prependImportStatements(module)` and have it do all the magic of creating the statements and appending them.